packages/manager/apps/dedicated/client/app/dedicated/server/dashboard/advanced-features/sgx/sgx.service.js

```diff
@@ -0,0 +1,19 @@
+import isString from 'lodash/isString';
+import toUpper from 'lodash/toUpper';
+
+import { DOCUMENTATION_BY_SUB } from './sgx.constants';
+
+export default class {
```

Comment:
**suggestion**: This service should be a plain object, there are only static methods in it.